lib/infrastructure/class-wp-http-response.php

```diff
@@ -167,9 +167,7 @@ public function set_data( $data ) {
 	 *
 	 * @return mixed Any JSON-serializable value.
 	 */
-	// @codingStandardsIgnoreStart
 	public function jsonSerialize() {
-	// @codingStandardsIgnoreEnd
```

Comment:
This one has to be ignored for WPCS, otherwise it throws a naming error, last I checked?